api.py

```diff
@@ -114,9 +152,20 @@ def verify_token():
     #If token is invalid...
     if result == "ERROR-INVALID-TOKEN":
         #Return an error
+
+        serviceutils.log_event("invalid_token", {
+                    "error": result,
+                    "false_token": token
+                    })
+
         return '{ "error": "' + result + '" }'
 
     #Otherwise, return the UID
+    serviceutils.log_event("token_valid", {
```

Comment:
I wouldn't log if the token is valid, because tokens get verified so often that we'd have a bunch of unimportant logs.

Reply:
Also, if the token _is not_ valid, I would also add the IP address of the computer sending the token to the data.

I'll add that later, though, so don't worry about it.